lib/ansible/modules/monitoring/zabbix_host.py

```diff
@@ -239,6 +247,11 @@ def update_host(self, host_name, group_ids, status, host_id, interfaces, exist_i
                 parameters['proxy_hostid'] = proxy_id
             if visible_name:
                 parameters['name'] = visible_name
+            if description:
+                parameters['description'] = description
+            else:
+                parameters['description'] = ""
```

Comment:
that's right.
but there is no way to unset the parameters (description, visible_name, ...).
you can only overwrite them.

how can i solve that problem?